doc/mgr/orchestrator_cli.rst

```diff
@@ -8,40 +8,167 @@ Orchestrator CLI
 
+As the orchestrator CLI unifies different external orchestrators, a common nomenclature
+for the orchrstrator module is needed.
+
++--------------------------------------+--------------------------------------+
+| host                                 | hostname (not DNS name) on the       |
+|                                      | physical host. Not the podname,      |
+|                                      | container name, or hostname inside   |
+|                                      | the container.                       |
++--------------------------------------+--------------------------------------+
+| service                              | A logical service, e.g., “MDS for    |
+|                                      | fs1” or “NFS gateway(s)”. Typically  |
+|                                      | comprised of multiple service        |
+|                                      | instances on multiple hosts for HA   |
++--------------------------------------+--------------------------------------+
+| service instance                     | A single instance of a service.      |
+|                                      |  Usually a daemon, but maybe not     |
+|                                      | (e.g., might be a kernel service     |
+|                                      | like LIO or knfsd or whatever)       |
++--------------------------------------+--------------------------------------+
+| daemon                               | A running process on a host; use     |
+|                                      | “service instance” instead           |
++--------------------------------------+--------------------------------------+
+
 Configuration
 =============
 
-You can select the orchestrator module to use with the ``set backend`` command:
-
-::
+You can select the orchestrator module to use with the ``set backend`` command::
 
     ceph orchestrator set backend <module>
 
-For example, to enable the Rook orchestrator module and use it with the CLI:
-
-::
+For example, to enable the Rook orchestrator module and use it with the CLI::
 
     ceph mgr module enable orchestrator_cli
     ceph mgr module enable rook
     ceph orchestrator set backend rook
 
 
-You can then check backend is properly configured:
-
-::
+You can then check backend is properly configured::
 
     ceph orchestrator status
 
 
 Usage
 =====
 
+.. warning::
+
+    The orchestrator CLI is unfinished and work in progress. Some commands will not
+    exist, or return a different result.
+
+.. note::
+
+    Orchestrator modules may only implement a subset of the commands listed below.
+    Also, the implementation of the commands are orchestrator module dependent and will
+    differ between implementations.
+
+Status
+~~~~~~
+
+::
+
+    ceph orchestrator status
+
+Show current orchestrator mode and high-level status (whether the module able
+to talk to it)
+
+Also show any in-progress actions.
+
+..
+
+    Host Management
```

Comment:
this is failing the docs build because it is indented (along with the underline)